test/Twig/Tests/EnvironmentTest.php

```diff
@@ -165,6 +173,9 @@ public function testExtensionsAreNotInitializedWhenRenderingACompiledTemplate()
         $this->assertEquals('bar', $output);
 
         unlink($cache);
+        if ($createdDir) {
+            rmdir($dir);
+        }
```

Comment:
alright, done in 27fcd6e53f5e40f757fac2cce5f9e4d073cf7163 :)
I'll squash later because the history on the branch is becoming a bit of a mess ;)